Core/include/Acts/Definitions/Direction.hpp

```diff
@@ -43,7 +47,7 @@ class Direction final {
   /// @return a direction enum
   static constexpr Direction fromScalar(double scalar) {
     assert(scalar != 0);
-    return scalar >= 0 ? Value::Positive : Value::Negative;
+    return Direction{scalar >= 0 ? Value::Positive : Value::Negative};
```

Comment:
I wonder if this would be easier
```suggestion
    return scalar >= 0 ? Positive() : Negative();
```